source/making_payments/index.html.md.erb

```diff
@@ -17,6 +17,8 @@ Your service will also need to supply a
 is a URL that your service hosts for your user to return to, after their
 payment journey on GOV.UK Pay ends.
 
+<%= warning_text('This does not apply if you only use <a href="https://www.payments.service.gov.uk/govuk-payment-pages/" target="_blank">payment links</a>.') %>
```

Comment:
should we add an intro to this section to say that this is about taking a payment using the API? (rather than having a warning about it not applying to payment links)? but also fine either way. Just feels like we jump into this page without a lot of context

Reply:
agree, have amended